templates/cisco_asa_show_version.template

```diff
@@ -19,9 +20,10 @@ Value LAST_MOD (.+)
 Start
   ^.*Software\sVersion\s${VERSION}
   ^Device.+\s${DEVICE_MGR_VERSION}
+  ^Compiled on \w+\s+${COMPILE_DATE}.*
```

Comment:
Can you use **\s*** or **\s+** instead of static whitespaces?

Reply:
👍